test/cli-test.js

```diff
@@ -91,10 +91,11 @@ test('cli - hash - no password', function (t){
   var stdin = execCli(['hash'], function (err, stdout, stderr){
     t.ifError(err);
 
-    var actual = stderr.trim();
-    var expected = '[Error: Password must be a  non-empty string.]';
+    var expected = /Error: Password must be a non-empty string/;
 
-    t.is(actual, expected);
+    t.throws(function (){
```

Comment:
We're still using the tape API when we check against the `stderr.message` property. Again, .throws() is for testing whether or not a function throws. Note how you had to create a fake function and test against that to use it... That just forces us to look harder to see what you're really trying to test...